charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -1080,6 +1080,13 @@ connectors:
 
 {{- define "opentelemetry-collector.kubernetesResourcesConfig" -}}
 processors:
+  {{- if .Values.presets.kubernetesResources.filterWorkflows.enabled }}
+  filter/workflow:
+    error_mode: silent
+    logs:
+      log_record:
+        - 'body["object"]["kind"] == "Pod" and not IsMatch(body["object"]["metadata"]["ownerReferences"], ".*StatefulSet.*|.*ReplicaSet.*|.*Job.*|.*DaemonSet.*")'
```

Comment:
Why don't we look for "workflow" records and ignore them instead of this "reverse logic" (only allow what is coming from statefulset, replicaset, etc)?